cross/iphone.txt

```diff
@@ -1,6 +1,6 @@
 # This is a cross compilation file from OSX Yosemite to iPhone
 # Apple keeps changing the location and names of files so
-# these might not work for you. Use the googels and xcrun.
+# these might not work for you. Use the googles and xcrun.
```

Comment:
Some of meson's code is cute, but I don't see any particular reason to misspell google here to be cute.